mungegithub/mungers/assign-reassign-handler.go

```diff
@@ -62,40 +63,54 @@ func (AssignReassignHandler) Initialize(config *github.Config, features *feature
 func (AssignReassignHandler) EachLoop() error { return nil }
 
 // AddFlags will add any request flags to the cobra `cmd`
-func (AssignReassignHandler) AddFlags(cmd *cobra.Command, config *github.Config) {}
+func (h AssignReassignHandler) AddFlags(cmd *cobra.Command, config *github.Config) {
+	cmd.Flags().BoolVar(&h.checkValid, "check-valid-reviewer", true, "Flag indicating whether to allow any one to be assigned or just OWNERs files reviewers")
+}
 
 // Munge is the workhorse the will actually make updates to the PR
 func (h AssignReassignHandler) Munge(obj *github.MungeObject) {
 	if !obj.IsPR() {
 		return
 	}
-	reviewers := getReviewers(obj)
-	if len(reviewers) == 0 {
-		return
-	}
 
 	comments, err := obj.ListComments()
 	if err != nil {
 		glog.Errorf("unexpected error getting comments: %v", err)
 		return
 	}
 
-	h.assignOrRemove(obj, comments, reviewers)
+	toAssign, toUnassign, err := h.assignOrRemove(obj, comments, true)
+	if err != nil {
+		return
+	}
+	//assign and unassign reviewers as necessary
+	for _, username := range toAssign.List(){
+		obj.AssignPR(username)
+	}
+	if len(toUnassign) > 0 {
+		is := goGithub.IssuesService{}
+		is.RemoveAssignees(OWNER, *obj.Issue.Repository.Name, *obj.Issue.Number, toUnassign.List())
```

Comment:
We usually go through the internal github layer to do that rather than use go-github directly. Hard-coding the OWNER is also a bad idea because it prevents people in other repo to use this code.